src/DataTypes/Serializations/SerializationDate.cpp

```diff
@@ -76,9 +76,22 @@ void SerializationDate::serializeTextCSV(const IColumn & column, size_t row_num,
 
 void SerializationDate::deserializeTextCSV(IColumn & column, ReadBuffer & istr, const FormatSettings &) const
 {
-    LocalDate value;
-    readCSV(value, istr);
-    assert_cast<ColumnUInt16 &>(column).getData().push_back(value.getDayNum());
+    DayNum x;
+
+    if (istr.eof())
+        throwReadAfterEOF();
+
+    char maybe_quote = *istr.position();
+
+    if (maybe_quote == '\'' || maybe_quote == '\"')
```

Comment:
There is a setting with name like 'Input format double quotes'

Reply:
you mean 'allow_single_quotes' and 'allow_double_quotes'? They were not used here before. They used only in readCSVString